src/interactive/modules.ts

```diff
@@ -0,0 +1,136 @@
+import * as vscode from 'vscode';
+import * as vslc from 'vscode-languageclient';
+import * as rpc from 'vscode-jsonrpc';
+import { onInit, onExit } from './repl'
+import { onSetLanguageClient } from '../extension';
+
+let statusBarItem: vscode.StatusBarItem = null
+let g_connection: rpc.MessageConnection = null
+let g_languageClient: vslc.LanguageClient = null
```

Comment:
I feel like explicitly `null`ing this is better because it's more explicit (and I think we're setting these to `null` e.g. when the connection is closed).

Reply:
Ok, lets leave for now, we should just come up with a consistent story across the code base at some point :)